packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -201,69 +200,128 @@ abstract class Scheduler extends BindingBase {
+    assert(() {
+      if (transientCallbackCount > 0) {
+        FlutterError.reportError(new FlutterErrorDetails(
+          exception: reason,
+          library: 'scheduler library',
+          informationCollector: (StringBuffer information) {
+            information.writeln(
+              'There ${ transientCallbackCount == 1 ? "was one transient callback" : "were $transientCallbackCount transient callbacks" } '
+              'left. The stack traces for when they were registered are as follows:'
+            );
+            for (int id in _transientCallbacks.keys) {
+              _FrameCallbackEntry entry = _transientCallbacks[id];
+              information.writeln('-- callback $id --');
+              information.writeln(entry.stack);
+            }
+          }
+        ));
+      }
+      return true;
+    });
+    return true;
+  }
+
+
   final List<FrameCallback> _persistentCallbacks = new List<FrameCallback>();
 
   /// Adds a persistent frame callback.
   ///
-  /// Persistent callbacks are invoked after transient (non-persistent) frame
-  /// callbacks.
+  /// Persistent callbacks are invoked after transient
+  /// (non-persistent) frame callbacks.
   ///
-  /// Does *not* request a new frame. Conceptually, persistent
-  /// frame-callbacks are thus observers of begin-frame events. Since they are
-  /// executed after the transient frame-callbacks they can drive the rendering
-  /// pipeline.
+  /// Does *not* request a new frame. Conceptually, persistent frame
+  /// callbacks are observers of "begin frame" events. Since they are
+  /// executed after the transient frame callbacks they can drive the
+  /// rendering pipeline.
   void addPersistentFrameCallback(FrameCallback callback) {
     _persistentCallbacks.add(callback);
   }
 
+
   final List<FrameCallback> _postFrameCallbacks = new List<FrameCallback>();
 
   /// Schedule a callback for the end of this frame.
   ///
   /// Does *not* request a new frame.
   ///
-  /// The callback is run just after the persistent frame-callbacks (which is
-  /// when the main rendering pipeline has been flushed). If a frame is
-  /// in progress, but post frame-callbacks haven't been executed yet, then the
-  /// registered callback is still executed during the frame. Otherwise,
-  /// the registered callback is executed during the next frame.
+  /// The callback is run just after the persistent frame callbacks
+  /// (which is when the main rendering pipeline has been flushed). If
+  /// a frame is in progress, but post frame callbacks haven't been
+  /// executed yet, then the registered callback is still executed
+  /// during the frame. Otherwise, the registered callback is executed
+  /// during the next frame.
   ///
-  /// The registered callbacks are executed in the order in which they have been
-  /// registered.
+  /// The registered callbacks are executed in the order in which they
+  /// have been registered.
   void addPostFrameCallback(FrameCallback callback) {
```

Comment:
Does the term "registered" in the text above correspond to added, as in addPostFrameCallback?